haiku-apps/bemobile/bemobile-0.9.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="A program to manage contacts and messages stored on your mobile phone"
+DESCRIPTION="BeMobile is a program to manage contacts and messages stored on your mobile phone. It has been developed for Motorola C550 and L6 models but should work with any phone that supports AT command set."
+HOMEPAGE="https://github.com/ytmytm/beos-bemobile"
+COPYRIGHT="2007 Maciej Witkowia"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev=
+SOURCE_URI="https://github.com/HaikuArchives/beos-mobile/archive/$srcGitRev.zip"
+CHECKSUM_SHA256="0006ae32ebbba2a51d59a4c29aa1fa03d4f7b69b9779b210885465d3238b7ce9"
+SOURCE_DIR="BeMobile-$srcGitRev"
```

Comment:
```diff
-srcGitRev=
+srcGitRev="fea6464ceb5b570f7a533e59b24a68f9fb093006"
-SOURCE_URI="https://github.com/HaikuArchives/beos-mobile/archive/$srcGitRev.zip"
+SOURCE_URI="https://github.com/ytmytm/beos-mobile/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="0006ae32ebbba2a51d59a4c29aa1fa03d4f7b69b9779b210885465d3238b7ce9"
+CHECKSUM_SHA256="2c903363e0725556d896ed5a15f68b77a0461b4512efd0106a2c68d8b30d4e5c"
+SOURCE_FILENAME="beos-bemobile-$portVersion-$srcGitRev.tar.gz"
-SOURCE_DIR="BeMobile-$srcGitRev"
+SOURCE_DIR="beos-bemobile-$srcGitRev"
```
(We prefer `tar.gz` over `zip`.)

Reply:
Of course, the srcGitRev comes from the "Latest commit" hyperlink in https://github.com/ytmytm/beos-bemobile.)